openquake/commands/plot_assets.py

```diff
@@ -75,6 +75,7 @@ def main(calc_id: int = -1, site_model=False,
             disc = numpy.unique(dstore['discarded']['lon', 'lat'])
             p.scatter(disc['lon'], disc['lat'], marker='x', color='red',
                       label='discarded', s=markersize_discarded)
+    min_x, max_x, min_y, max_y = (180, -180, 90, -90)
```

Comment:
I would have expected (-180, 180, -90, 90) ?

Reply:
This way, for instance, `min_x` is initially set to its upper limit, and afterwards we get the minimum between that and a computed one.